crates/build/re_build_tools/Cargo.toml

```diff
@@ -24,6 +24,7 @@ anyhow = { workspace = true, default-features = true }
 cargo_metadata.workspace = true
 glob.workspace = true
 sha2.workspace = true
-time = { workspace = true, features = ["formatting"] }
+jiff = { workspace = true }
 unindent.workspace = true
 walkdir.workspace = true
+regex = "1.0"
```

Comment:
A few points here:

A) you're only using regex for tests, so this should be under `[dev-dependencies]`
B) all dependencies should be declared in the workspace
C) for faster compile times, please switch to [`regex-lite`](https://crates.io/crates/regex-lite)
D) We should set the initial version string to what's in Cargo.lock, otherwise Cargo might pick up the `1.0.0` version, which might be missing some features we're using